Python/basicput.py

```diff
@@ -52,11 +52,13 @@ def connect():
         if MQDetails[EnvStore.KEY_REPOSITORY]:
```

Comment:
Other than an extra print, this file has not changed and should not be part or the pull request.